views/src/app/Auth/components/editor/editor.js

```diff
@@ -1,42 +1,42 @@
-import React, { Component } from 'react';
+import React, { useState } from "react";
 import axios from 'axios';
 import 'suneditor/dist/css/suneditor.min.css';
 import SunEditor, { buttonList, } from "suneditor-react";
-import { Container, Col, Row } from 'react-bootstrap';
+import { Container, Col, Row, Button } from 'react-bootstrap';
+import { useHistory } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './editor.css';
 
-class MyComponent extends Component {
-    constructor(props) {
-        super(props);
-        this.state = { title: "", content: "" };
-    }
+const PostEditorComponent = () => {
+  
+    const [title, setTitle] = useState("");
+    const [content, setContent] = useState("");
 
-    handleChange = (content) => {
-        this.setState({ content });
-    }
+    const history = useHistory();
 
-    getTitle = (event) => {
-        let title = event.target.value;
-        this.setState({ title });
+    let handlePost = (postId) => {
+        let path = `/postWithComments/${postId}`
+        history.push(path);
     }
 
-    handleFocus = event => {
+    let handleFocus = event => {
         console.log(event);
     }
 
-    handleBlur = event => {
+    let handleBlur = event => {
         console.log(event);
     }
 
-    sendPost = async () => {
+    let sendPost = async () => {
         try {
-            console.log("TOKEN", localStorage);
             const res = await axios.post(`http://localhost:3001/api/posts/`,
-                { data: this.state, userId: localStorage.userId },
+                { data: { title, content }, userId: localStorage.userId },
```

Comment:
you shouldn't need to send the `userId` from local storage since you can easily get it on the server in the token

Reply:
That's a good idea, thank you, will do!